client/gutenberg/extensions/presets/jetpack/package.json

```diff
@@ -10,8 +10,9 @@
 		"access": "public"
 	},
 	"scripts": {
+		"clean": "rm -fr ./build",
```

Comment:
Just noting that this isn't Windows friendly but I'm okay with that since this isn't the main `package.json` for Calypso.

If we use Lerna more extensively, this might come back to us, though.